lib/datasource/docker/common.spec.ts

```diff
@@ -1,3 +1,4 @@
+import nock from 'nock';
```

Comment:
```suggestion
import * as httpMock from '../../../test/http-mock';
```
use out http test util, otherwise we get memory issues from nock.